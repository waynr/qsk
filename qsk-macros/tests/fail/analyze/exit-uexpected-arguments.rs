use qsk_macros::remap;

fn main() {
    remap!(
        ModLayer: {
            F -> Exit(MEOW),
        },
    );
}
