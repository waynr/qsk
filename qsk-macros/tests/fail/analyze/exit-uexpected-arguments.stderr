error: unexpected argument
 --> tests/fail/analyze/exit-uexpected-arguments.rs:6:23
  |
6 |             F -> Exit(MEOW),
  |                       ^^^^
